src/EditorFeatures/CSharp/Formatting/Indentation/SmartTokenFormatterCommandHandler.cs

```diff
@@ -1,63 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using System.ComponentModel.Composition;
-using System.Linq;
-using System.Threading;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.Editor.Implementation.Formatting.Indentation;
-using Microsoft.CodeAnalysis.Formatting;
-using Microsoft.CodeAnalysis.Formatting.Rules;
-using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.VisualStudio.Text.Operations;
-using Microsoft.VisualStudio.Utilities;
-using Roslyn.Utilities;
-using VSCommanding = Microsoft.VisualStudio.Commanding;
-
-namespace Microsoft.CodeAnalysis.Editor.CSharp.Formatting.Indentation
-{
-    [Export(typeof(VSCommanding.ICommandHandler))]
-    [ContentType(ContentTypeNames.CSharpContentType)]
-    [Name(PredefinedCommandHandlerNames.Indent)]
-    [Order(After = PredefinedCommandHandlerNames.Rename)]
-    [Order(Before = PredefinedCommandHandlerNames.Completion)]
-    internal class SmartTokenFormatterCommandHandler :
-        AbstractSmartTokenFormatterCommandHandler
```

Comment:
This type is completely unnecessary.  I don't know why we had it.  We already exposed the ISmartIndentProvider VS type to expose this functionality.  So all this code was removed and integrated into IIndentationService.